CreditManagement/views.py

```diff
@@ -96,20 +105,28 @@ def post(self, request, *args, **kwargs):
         context['form'] = form
         return self.render_to_response(context)
 
+    def check_access_permission(self, request):
+        if self.transaction_obj is not None:
+            if self.transaction_obj.confirm_moment <= timezone.now():
+                return HttpResponseForbidden("This transaction can no longer be altered")
+        return super(TransactionBaseAlterView, self).check_access_permission(request)
+
 
-class TransactionUserView(LoginRequiredMixin, TransactionAlterView):
-    def get_form(self, transaction=None, data=None):
+class TransactionUserView(LoginRequiredMixin, TransactionBaseAlterView):
+    """
+    Basic user transaction view
```

Comment:
I've learned to be using a standard consistent coding style as well as a consistent docstring style at my summer job where we're also using Django, and it makes the code much nicer and homogeneous. I want to apply this here as well (#157)

For docstring style I want to be following the Google style guide ([here](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)). I've compared the different styles that there are out there and this is a clear winner over the default style used by PyCharm. You can change the docstring format to Google's in PyCharm settings.

I would suggest that if you add comments that you should make them meaningful and not just repeat the name of the class.

Reply:
I'll try to use it in my future works.